docs/bcfusion/validators/migrations.md

```diff
@@ -1,2 +1,11 @@
-This page will guide validators to migrate their BSC validators.
-Please be patient for updates.
\ No newline at end of file
+# Validator Migration
+
+The [native staking](https://github.com/bnb-chain/BEPs/blob/master/BEPs/BEP294.md)
```

Comment:
Please only suggest this method when the old validator would like to use cli and always guide them to use the dapp.